girder/molecules/server/calculation.py

```diff
@@ -267,8 +267,7 @@ def create_calc(self, params):
             file_id = file['_id']
             cjson, file_str = self._file_to_cjson(file, code)
 
-            if 'vibrations' in cjson or 'basisSet' in cjson:
-                props = self._extract_calculation_properties(cjson, file_str, code)
```

Comment:
Why has this guard been removed?

Reply:
It wasn't 100% clear to me why it was there in the first place. The reason why I removed it now, is that when we ingest a ML calculation `vibrations` and `basisSet` will not be in the parsed cjson